zeek-config.h.in

```diff
@@ -149,6 +149,8 @@
 // For example, 3.1.2 becomes 30102.
 #define ZEEK_VERSION_NUMBER @ZEEK_VERSION_NUMBER@
 
+#define ZEEK_BUILD_INFO_ESCAPED "@ZEEK_BUILD_INFO_ESCAPED@"
```

Comment:
Sorry for being annoying (and I'm feeling bad as I caused this). This will break ccache workflows as anything that somehow includes zeek-config.h will need to be rebuild on every commit/configure cycle.

It's essentially https://github.com/zeek/zeek/issues/2776 made worse because ZEEK_BUILD_INFO changes whenever commit+configure runs.

Very naively: Do we maybe need to change the order of src/obj files so that version.cc comes last here (like when linking objects the order needs to be "right")
https://github.com/zeek/zeek/blob/8e2d68ffecb0384747275e7d05c9b75729f4750b/src/CMakeLists.txt#L514-L536

I'm okay moving this forward, but will then stare at #2776, but I'm not sure it's clear how to untangle this :-(

Reply:
Fixed by using `extern "C"` instead in zeek-setup.cc (caused because version.c is built a C object and not C++ and MSV appears sensitive).